### ISSUE TITLE
windows_task resource issue with 13.6.4 - undefined method `elements' for nil:NilClass

### ISSUE THREAD

**Issue by croffe** [chef/chef#6585]
## Description

windows_task error with 13.6.4 - undefined method `elements' for nil:NilClass

## Chef Version

13.6.4

## Platform Version

Windows Server 2008 (6.0.6002)

## Replication Case

Leverage chef-client supermarket library to create Windows Task to manage Chef Client

## Client Output

Recipe: chef-client::task
  * windows_service[chef-client] action configure_startup (skipped due to only_if)
  * chef_client_scheduled_task[Chef Client] action add
    * windows_task[chef-client] action create

      ================================================================================
      Error executing action `create` on resource 'windows_task[chef-client]'
      ================================================================================

      NoMethodError
      -------------
      undefined method `elements' for nil:NilClass

      Resource Declaration:
      ---------------------
      # In C:/chef/cache/cookbooks/chef-client/resources/scheduled_task.rb

       46:   windows_task 'chef-client' do
       47:     run_level :highest
       48:     command "cmd /c \"#{client_cmd}\""
       49:
       50:     user               new_resource.user
       51:     password           new_resource.password
       52:     frequency          new_resource.frequency.to_sym
       53:     frequency_modifier new_resource.frequency_modifier
       54:     start_time         new_resource.start_time || start_time
       55:   end
       56: end

      Compiled Resource:
      ------------------
      # Declared in C:/chef/cache/cookbooks/chef-client/resources/scheduled_task.rb:46:in `block in class_from_file'

      windows_task("chef-client") do
        action :create
        default_guard_interpreter :default
        task_name "chef-client"
        declared_type :windows_task
        cookbook_name "chef-client"
        run_level :highest
        command "cmd /c \"C:/opscode/chef/bin/chef-client -L C:/chef/log/client.log -c C:/chef/client.rb -s 300\""
        user "SYSTEM"
        password nil
        frequency :minute
        frequency_modifier 60
        start_time "09:04"
        execution_time_limit "PT72H"
      end

      System Info:
      ------------
      chef_version=13.6.4
      platform=windows
      platform_version=6.0.6002
      ruby=ruby 2.4.2p198 (2017-09-14 revision 59899) [x64-mingw32]
      program_name=C:/opscode/chef/bin/chef-client
      executable=C:/opscode/chef/bin/chef-client

    ================================================================================
    Error executing action `add` on resource 'chef_client_scheduled_task[Chef Client]'
    ================================================================================

    NoMethodError
    -------------
    windows_task[chef-client] (C:/chef/cache/cookbooks/chef-client/resources/sch
eduled_task.rb line 46) had an error: NoMethodError: undefined method `elements'
 for nil:NilClass

    Resource Declaration:
    ---------------------
    # In C:/chef/cache/cookbooks/chef-client/recipes/task.rb

     37: chef_client_scheduled_task 'Chef Client' do
     38:   user node['chef_client']['task']['user']
     39:   password node['chef_client']['task']['password']
     40:   frequency node['chef_client']['task']['frequency']
     41:   frequency_modifier lazy { node['chef_client']['task']['frequency_modifier'] }
     42:   start_time node['chef_client']['task']['start_time']
     43:   splay node['chef_client']['splay']
     44:   config_directory node['chef_client']['conf_dir']
     45:   log_directory node['chef_client']['log_dir']
     46:   chef_binary_path node['chef_client']['bin']
     47:   daemon_options node['chef_client']['daemon_options']
     48: end

    Compiled Resource:
    ------------------
    # Declared in C:/chef/cache/cookbooks/chef-client/recipes/task.rb:37:in `fro
m_file'

    chef_client_scheduled_task("Chef Client") do
      action [:add]
      default_guard_interpreter :default
      declared_type :chef_client_scheduled_task
      cookbook_name "chef-client"
      recipe_name "task"
      user "SYSTEM"
      password nil
      frequency "minute"
      frequency_modifier 60
      start_time nil
      splay "300"
      config_directory "C:/chef"
      log_directory "C:/chef/log"
      chef_binary_path "C:/opscode/chef/bin/chef-client"
      daemon_options []
    end

    System Info:
    ------------
    chef_version=13.6.4
    platform=windows
    platform_version=6.0.6002
    ruby=ruby 2.4.2p198 (2017-09-14 revision 59899) [x64-mingw32]
    program_name=C:/opscode/chef/bin/chef-client
    executable=C:/opscode/chef/bin/chef-client

Running handlers:
[2017-11-15T08:04:02-07:00] ERROR: Running exception handlers
Running handlers complete
[2017-11-15T08:04:02-07:00] ERROR: Exception handlers complete
Chef Client failed. 0 resources updated in 20 seconds
[2017-11-15T08:04:03-07:00] FATAL: Stacktrace dumped to C:/chef/cache/chef-stack
trace.out
[2017-11-15T08:04:03-07:00] FATAL: Please provide the contents of the stacktrace
.out file if you file a bug report
[2017-11-15T08:04:03-07:00] FATAL: NoMethodError: chef_client_scheduled_task[Che
f Client] (chef-client::task line 37) had an error: NoMethodError: windows_task[
chef-client] (C:/chef/cache/cookbooks/chef-client/resources/scheduled_task.rb li
ne 46) had an error: NoMethodError: undefined method `elements' for nil:NilClass

## Stacktrace

Generated at 2017-11-15 08:04:03 -0700
NoMethodError: chef_client_scheduled_task[Chef Client] (chef-client::task line 37) had an error: NoMethodError: windows_task[chef-client] (C:/chef/cache/cookbooks/chef-client/resources/scheduled_task.rb line 46) had an error: NoMethodError: undefined method `elements' for nil:NilClass
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider/windows_task.rb:392:in `load_task_xml'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider/windows_task.rb:373:in `load_task_hash'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider/windows_task.rb:37:in `load_current_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider.rb:154:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource.rb:591:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:70:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block (2 levels) in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `each'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:94:in `block in execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:114:in `call_iterator_block'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:85:in `step'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:103:in `iterate'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:55:in `each_with_index'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:92:in `execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:97:in `converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider.rb:212:in `compile_and_converge_action'
(eval):2:in `action_add'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider.rb:171:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource.rb:591:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:70:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block (2 levels) in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `each'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:94:in `block in execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:114:in `call_iterator_block'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:85:in `step'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:103:in `iterate'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:55:in `each_with_index'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:92:in `execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:97:in `converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:718:in `block in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:713:in `catch'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:713:in `converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:752:in `converge_and_save'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:286:in `run'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:273:in `run_with_graceful_exit_option'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:249:in `block in run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/local_mode.rb:44:in `with_server_connectivity'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:232:in `run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:469:in `sleep_then_run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:458:in `block in interval_run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:457:in `loop'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:457:in `interval_run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:441:in `run_application'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:59:in `run'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/bin/chef-client:26:in `<top (required)>'
C:/opscode/chef/bin/chef-client:58:in `load'
C:/opscode/chef/bin/chef-client:58:in `<main>'

>>>> Caused by NoMethodError: undefined method `elements' for nil:NilClass
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider/windows_task.rb:392:in `load_task_xml'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider/windows_task.rb:373:in `load_task_hash'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider/windows_task.rb:37:in `load_current_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider.rb:154:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource.rb:591:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:70:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block (2 levels) in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `each'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:94:in `block in execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:114:in `call_iterator_block'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:85:in `step'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:103:in `iterate'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:55:in `each_with_index'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:92:in `execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:97:in `converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider.rb:212:in `compile_and_converge_action'
(eval):2:in `action_add'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/provider.rb:171:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource.rb:591:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:70:in `run_action'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block (2 levels) in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `each'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:98:in `block in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:94:in `block in execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:114:in `call_iterator_block'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:85:in `step'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:103:in `iterate'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/stepable_iterator.rb:55:in `each_with_index'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/resource_collection/resource_list.rb:92:in `execute_each_resource'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/runner.rb:97:in `converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:718:in `block in converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:713:in `catch'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:713:in `converge'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:752:in `converge_and_save'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/client.rb:286:in `run'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:273:in `run_with_graceful_exit_option'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:249:in `block in run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/local_mode.rb:44:in `with_server_connectivity'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:232:in `run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:469:in `sleep_then_run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:458:in `block in interval_run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:457:in `loop'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:457:in `interval_run_chef_client'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application/client.rb:441:in `run_application'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/lib/chef/application.rb:59:in `run'
C:/opscode/chef/embedded/lib/ruby/gems/2.4.0/gems/chef-13.6.4-universal-mingw32/bin/chef-client:26:in `<top (required)>'
C:/opscode/chef/bin/chef-client:58:in `load'
C:/opscode/chef/bin/chef-client:58:in `<main>'

### NOTE: CHEF CLIENT BUGS ONLY

This issue tracker is for the code contained within this repo -- `chef-client`, base `knife` functionality (not
plugins), `chef-apply`, `chef-solo`, `chef-client -z`, etc.

* Requests for new or alternative functionality should be made to [feedback.chef.io](https://feedback.chef.io/forums/301644-chef-product-feedback/category/110832-chef-client)
* [Chef Server issues](https://github.com/chef/chef-server/issues/new)
* [ChefDK issues](https://github.com/chef/chef-dk/issues/new)
* Cookbook Issues (see the https://github.com/chef-cookbooks repos or search [Supermarket](https://supermarket.chef.io) or GitHub/Google)

**Comment by lamont-granquist** [comment]
@tas50 is this a dup of known issues that are being fixed?

**Comment by sean-horn** [comment]
This is Zendesk 16646 if needed for future reference

**Comment by NAshwini** [comment]
Hi @croffe, @sean-horn, 
I have tried the same recipe with the latest code present in chef git repo: https://github.com/chef/chef.git and was not able to reproduce the issue. The recipe is working fine by creating a scheduled task with minute frequency. This issue might be resolved with another issue. 
Tried with below recipes & its working fine.
1. 
```
windows_task 'test2' do
  run_level :highest
  command "cmd /c "#{client_cmd}""
  user 'SYSTEM'
  password nil
  frequency :minute
  frequency_modifier 60
  start_time "06:10"
end
```
2. 
```
windows_task 'test1' do
  run_level :highest
  command "cmd /c "#{client_cmd}""
  user 'Administrator'
  password 'xxx@123'
  frequency :minute
  frequency_modifier 60
  start_time "06:10"
end
```
system Information:
```
PS D:\> chef -v
Chef Development Kit Version: 2.4.22
chef-client version: 13.6.4
delivery version: master (73ebb72a6c42b3d2ff5370c476be800fee7e5427)
berks version: 6.3.1
kitchen version: 1.19.2
inspec version: 1.49.2
PS D:\> ruby -v
ruby 2.4.2p198 (2017-09-14 revision 59899) [x64-mingw32]
```
Please let me know in case of any query or concern.
Thank  You.

**Comment by tas50** [comment]
It looks like we've fixed this one with the latest changed that shipped in 13.7. @croffe if you're still seeing this in 13.7 please open this up and ping me so I can dig in further.
